script.js

```diff
@@ -0,0 +1,768 @@
+var board = [[0, 0, 0], [0, 0, 0], [0, 0, 0]];
+var round = 0;
+var computerplay = false;
```

Comment:
Declaring all variables at the top, as well as representing data in JavaScript instead of through DOM elements, are both recommended. Great job!